src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -221,6 +222,21 @@
           "while",
           "with");
 
+  /**
+   * Incremental clutz cannot infer class aliases like:
+   * <pre>
+   *   @constructor
```

Comment:
This really should be /** @constructor */
which can't work in this comment but maybe you can avoid the "*/" part of it

Reply:
done